contracts/Safe.sol

```diff
@@ -175,7 +175,14 @@ contract Safe is
             gasUsed = gasUsed.sub(gasleft());
             // If no safeTxGas and no gasPrice was set (e.g. both are 0), then the internal tx is required to be successful
             // This makes it possible to use `estimateGas` without issues, as it searches for the minimum gas where the tx doesn't revert
-            if (!success && safeTxGas == 0 && gasPrice == 0) revertWithError("GS013");
+            if (!success && safeTxGas == 0 && gasPrice == 0) {
+                /* solhint-disable no-inline-assembly */
+                assembly {
+                    returndatacopy(0, 0, returndatasize())
+                    revert(0, returndatasize())
+                }
+                /* solhint-enable no-inline-assembly */
```

Comment:
wdyt about doing it in a memory-safe way?
![Screenshot 2024-09-30 at 10 05 53](https://github.com/user-attachments/assets/88ba6f65-5a03-41a8-a3ef-e8a9014060d6)
From the "Memory Safety" section in https://docs.soliditylang.org/en/latest/assembly.html

Reply:
Done 👍🏾